content/chainguard/chainguard-images/how-to-use/verifying-chainguard-images-and-metadata-signatures-with-cosign/index.md

```diff
@@ -21,20 +21,57 @@ toc: true
 
 ## Registry and Tags for Chainguard Containers
 
 Attestations are provided per image build, so you'll need to specify the correct tag and registry when pulling attestations from an image with `cosign`.
 
-- `cgr.dev/chainguard` - the Public Registry contains our **Starter container images**, which typically comprise the `:latest` versions of an image.
-- `cgr.dev/your-registry-name` - A Private/Dedicated Registry contains our **Production container images**, which include all versioned tags of an image and special images that are not available in the public registry (including FIPS images and other custom builds).
+- `cgr.dev/chainguard`: The Public Registry contains Chainguard's **Starter container images**, which typically comprise the `:latest` versions of an image.
+- `cgr.dev/YOUR-ORGANIZATION`: A Private/Dedicated Registry contains your organization's **Production container images**, which include all versioned tags of an image and special images that are not available in the public registry (including FIPS images and other custom builds).
 
 The commands listed on this page will default to the `:latest` tag, but you can specify a different tag to fetch attestations for.
 
+
+## Chainguard's Signing Identities
+
+Chainguard uses an identity associated with its official GitHub account to sign images in the public registry that contains the free tier of Starter images.
+
+For private images, Chainguard signs all images in your private registry with one of two different identities in your organization:
+
+* The `catalog_syncer` identity is used to sign images that have been imported directly from the Chainguard Containers catalog. 
+* The `apko_builder` identity is used to sign any images that have been customized for your organization, such as those built with [Custom Assembly](/chainguard/chainguard-images/features/ca-docs/custom-assembly/).
+
+These identities are created and added to every [verified Chainguard organization](/chainguard/administration/iam-organizations/verified-orgs/) automatically. 
+
+To follow along with the **Private Registry** examples in this guide, you will need the UIDPs of these Chainguard identities. To this end, create a few environment variables, the first of which should point to the name of your Chainguard organization:
+
+```shell
+PARENT=your-organization
+```
+
+Next, use this variable to create two more to hold the UIDPs of your organization's `catalog_syncer` and `apko_builder` identities, respectively:
+
+```shell
+CATALOG_SYNCER=$(chainctl iam account-associations describe $PARENT -o json | jq -r '.[].chainguard.service_bindings.CATALOG_SYNCER')
+APKO_BUILDER=$(chainctl iam account-associations describe $PARENT -o json | jq -r '.[].chainguard.service_bindings.APKO_BUILDER')
+```
+
+Some of the **Private Registry** examples in this guide will include these environment variables, allowing you to verify that these identities did indeed sign the given image. 
```

Comment:
Consider dropping "Some of"

Reply:
Could also change the second part to "and verify that the identities were used to sign the given image"